From: Stanislav Fomichev <sdf@google.com>
Date: Mon, 6 Jul 2020 16:01:27 -0700
Subject: bpftool: Add support for BPF_CGROUP_INET_SOCK_RELEASE
Patch-mainline: v5.9-rc1
Git-commit: db94cc0b4805968a0357ed2507730cdf77adf174
References: bsc#1177028

Support attaching to BPF_CGROUP_INET_SOCK_RELEASE and properly
display attach type upon prog dump.

Signed-off-by: Stanislav Fomichev <sdf@google.com>
Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>
Acked-by: Andrii Nakryiko <andriin@fb.com>
Link: https://lore.kernel.org/bpf/20200706230128.4073544-4-sdf@google.com
Acked-by: Gary Lin <glin@suse.com>
---
 tools/bpf/bpftool/common.c |    1 +
 1 file changed, 1 insertion(+)

--- a/tools/bpf/bpftool/common.c
+++ b/tools/bpf/bpftool/common.c
@@ -33,6 +33,7 @@ const char * const attach_type_name[__MA
 	[BPF_CGROUP_INET_INGRESS]	= "ingress",
 	[BPF_CGROUP_INET_EGRESS]	= "egress",
 	[BPF_CGROUP_INET_SOCK_CREATE]	= "sock_create",
+	[BPF_CGROUP_INET_SOCK_RELEASE]	= "sock_release",
 	[BPF_CGROUP_SOCK_OPS]		= "sock_ops",
 	[BPF_CGROUP_DEVICE]		= "device",
 	[BPF_CGROUP_INET4_BIND]		= "bind4",
